src/frontend/src/pages/HomePage/OnboardingHomePage.tsx

```diff
@@ -6,19 +6,23 @@ import LoadingIndicator from '../../components/LoadingIndicator';
 import ErrorPage from '../ErrorPage';
 import { useHomePageContext } from '../../app/HomePageContext';
 import InfoBlock from './components/InfoBlock';
+import { useAllUsefulLinks } from '../../hooks/projects.hooks';
 
 const OnboardingHomePage = () => {
   const { data: organization, isError, error, isLoading } = useCurrentOrganization();
   const { setOnPNMHomePage, setOnGuestHomePage, setOnOnboardingHomePage } = useHomePageContext();
 
+  const { data: links, isError: linksIsError, error: linksError, isLoading: linksIsLoading } = useAllUsefulLinks();
```

Comment:
Instead of calling this hook here and then passing in links to the infoBlock, can you just call this hook in that file itself so you don't need to pass in the links prop. And then you can call the loading page and error page from there as well